src/crispy/sse2neon.h

```diff
@@ -0,0 +1,8801 @@
+#ifndef SSE2NEON_H
+#define SSE2NEON_H
+
```

Comment:
```suggestion
// This file was taken from: https://github.com/JishinMaster/simd_utils/blob/master/sse2neon.h

```
So keeping this file up-to-date with regards to bugfixes but also giving credit, is easier :)